src/toolbar.scss

```diff
@@ -163,6 +163,16 @@ $title-toolbar-height: 2.75rem;
           margin: 0.375rem 0.1875rem;
           width: auto;
           background-color: var(--sapToolbar_SeparatorColor);
+          &+ .#{$fd-namespace}-button {
```

Comment:
including fd-button inside fd-toolbar is violating the self-contained rules. If inside your component (in this case fd-toolbar) you are overwriting the properties of another component (fd-button in this case) you can't target directly the element like you did, you need a modifier class. 
Basically you will need something like: `fd-toolbar__button` where you will modify the specific property of `fd-button`. But the problem here is that the visual specs do not specify spacing for the buttons in the overflow menu. The button is supposed to have "touchable area" that is bigger than the dimensions of the button and this touchable area is playing the role of "margin" around the button

Reply:
From the specs:
<img width="618" alt="Screen Shot 2021-03-12 at 3 26 30 PM" src="https://user-images.githubusercontent.com/39598672/110995449-60cf1480-8348-11eb-8b78-16bb8113b7d0.png">

Reply:
@InnaAtanasova I think it may be safe to assume the same spacing as buttons in the action sheet menu?

Reply:
> including fd-button inside fd-toolbar is violating the self-contained rules. If inside your component (in this case fd-toolbar) you are overwriting the properties of another component (fd-button in this case) you can't target directly the element like you did, you need a modifier class.

we can apply the same spacing for all elements in the menu, that way we should remove some of classes like

.#{$block}__overflow__label {
          &.#{$fd-namespace}-label {
                 margin: 0.375rem 0;
          

> Basically you will need something like: `fd-toolbar__button` where you will modify the specific property of `fd-button`. But the problem here is that the visual specs do not specify spacing for the buttons in the overflow menu. The button is supposed to have "touchable area" that is bigger than the dimensions of the button and this touchable area is playing the role of "margin" around the button

I see now but why margin around the button has to do with "touchable area" by default padding is doing that?